spec/lib/chip/service_spec.rb

```diff
@@ -560,7 +560,7 @@
       end
 
       it 'returns 200 with success message' do
-        VCR.use_cassette('chip/authenticated_demographics/get_demographics_200', match_requests_on: [:host]) do
```

Comment:
I was hoping not to change these at all, but if you go with the ERB approach, the ERB tags are necessary. I also think that it's probably better to match on the full route rather than on just the host.